File: C/Exercice3_2.c
#include <stdlib.h>
#include <stdio.h>

int main(int argc, char* argv[])
{
    char commande[256];

    if(argc == 1)
    {
        printf("Vous n'avez entrer aucun arguments\n");
        exit(0);
    }
    else if(argc >= 2)
    {
        snprintf(commande, sizeof(commande), "cp -r ../Fichier/ %s", argv[1]);
        system(commande);
        printf("Le fichier %s a ete copie avec succes\n");
    }
    return 0;
}

File: Python/Exercice3_2.py
#coding: utf-8
import sys
import subprocess

if len(sys.argv) == 1:
    print("Vous n'avez entrer aucun arguments")
    sys.exit(0)

else:
    commande = f"cp -r ../Fichier/ {sys.argv[1]}"
    subprocess.run(commande, shell=True)
    print("Le fichier a ete copie avec succes")